Dart/resources/META-INF/dart-yaml.xml

```diff
@@ -5,7 +5,7 @@
                      enabledByDefault="true" level="WARNING" language="yaml"
                      implementationClass="com.jetbrains.lang.dart.ide.inspections.DartPathPackageReferenceInspection"/>
     <psi.referenceContributor language="yaml" implementation="com.jetbrains.lang.dart.psi.PubspecYamlReferenceContributor" order="last"/>
-    <fileType name="YAML" fileNames=".analysis_options"/>
+    <fileType name="YAML" fileNames=".analysis_options;analysis_options.yaml;pubspec.yaml;fix_data.yaml"/>
```

Comment:
I don' think we need to list `*.yaml` files here, they already match standard `*.yaml` file pattern. Have you faced any issues without this change?

Reply:
No... doing a double-take after your comment, PR updated.